arch/arm/core/aarch64/CMakeLists.txt

```diff
@@ -20,3 +20,4 @@ zephyr_library_sources(
 
 zephyr_library_sources_ifdef(CONFIG_GEN_SW_ISR_TABLE isr_wrapper.S)
 zephyr_library_sources_ifdef(CONFIG_IRQ_OFFLOAD irq_offload.c)
+zephyr_library_sources_ifdef(CONFIG_ARM_MMU arm_mmu.c)
```

Comment:
Do you anticipate the need of a sub-directory here instead of a single MMU driver source file?

Reply:
I do not anticipate need of sub-directory as of now; Especially, no vendor specific mmu code(I see some for cortex-m). We can always add sub-directory later if need arises.

Reply:
ok